client/src/components/LinkField/LinkField.js

```diff
@@ -52,6 +52,7 @@ const LinkField = ({
   ownerID,
   ownerClass,
   ownerRelation,
+  excludeTextField = false,
```

Comment:
```suggestion
  excludeLinkTextField = false,
```
It's really confusing omitting the `Link` from this variable, especially since in other places it's not omitted.  It's a specific field we want to remove, not just a generic 'textfield'

Update all variables through including the ones passed to jquery as an element attribute e.g. 'data-exclude-textfield' should be 'data-exclude-linktext-field

Reply:
Done